module_utils/ns1.py

```diff
@@ -10,10 +10,10 @@
 
 from ansible.module_utils.basic import AnsibleModule
 
-try: 
-    from ansible.module_utils.basic import missing_required_lib 
-except Exception: 
-    def missing_required_lib(msg, reason=None, url=None): 
+try:
+    from ansible.module_utils.basic import missing_required_lib
+except Exception:
```

Comment:
this is just whitespace cleanup, but can we make this `except ImportError`?

Reply:
Because the code below can make `missing_required_lib()` work regardless of whether the module imports correctly or not, wouldn't our code be more robust if we override it if _any_ import time exception occurs, not just `ImportError`?